docs/configuration.rst

```diff
@@ -493,27 +493,27 @@ The KSQL Server image uses variables prefixed with ``KSQL_`` with an underscore
     -e KSQL_KSQL_SERVICE_ID=ksql_test_service_id_ \
     -e KSQL_KSQL_STREAMS_REPLICATION_FACTOR=3 \
     -e KSQL_KSQL_SINK_REPLICAS=3 \
-    -e KSQL_KSQL_STREAMS_STATE_DIR=/mnt/data/ksql-server-data \
-    -v /mnt/ksql-server/data:/mnt/data/ksql-server-data/ \
+    -e KSQL_KSQL_STREAMS_STATE_DIR=/ \
+    -v /docker/host/path:/docker/container/path \
     confluentinc/cp-ksql-server:4.1.2
 
 Docker Options
 """"""""""""""
 
-* Data persistence: KSQL Server runs Kafka streams applications which may need to store persistent state. The command above maps the state store directory to a directory on the localhost, hence ensuring persistence across multiple sessions of the same container.
-* Port mapping: The above command maps the listener port to 8088 on the localhost, hence ensuring that the CLI running on the same host can connect to the ksql server.
+* Data persistence: KSQL Server runs Kafka Streams applications which may need to store persistent state locally. The command above maps the state store directory inside the container (cf. KSQL_KSQL_STREAMS_STATE_DIR) to a directory on the Docker host.
+* Port mapping: The above command maps the listener port to 8088 on the Docker host, hence ensuring that other containers (like for the KSQL CLI) running on the same host can connect to the KSQL server container.
 
 Required Settings
 """""""""""""""""
 The following settings must be passed to run the KSQL Server image.
 
 ``KSQL_BOOTSTRAP_SERVERS``
 
-  A list of host/port pairs to use for establishing the initial connection to the Kafka cluster. The client will make use of all servers irrespective of which servers are specified here for bootstrapping; this list only impacts the initial hosts used to discover the full set of servers. This list should be in the form host1:port1,host2:port2,.... Since these servers are just used for the initial connection to discover the full cluster membership (which may change dynamically), this list need not contain the full set of servers (you may want more than one, though, in case a server is down).
+To configure the Kafka cluster that KSQL should read from/write to, you must specify one or more "bootstrap" brokers from that Kafka cluster. This parameter is formatted as a comma-separated list of hostname:port pairs; for example, broker-hostname1:9092,broker-hostname2:9092. These brokers are used by KSQL to establish the initial connection to the Kafka cluster and to discover the set of all available brokers in the Kafka cluster, the set of which may change dynamically at runtime (e.g. brokers may be taken down for maintenance). You can but don't need to specify all brokers in the Kafka cluster, though it is recommended to specify at least two brokers to make bootstrapping more resilient against individual broker outages.
```

Comment:
I'd suggest to use backticks around `broker-hostname1:9092,broker-hostname2:9092` to make it clear what of the paragraph text is the config value.